docs/v1/P4Runtime-Spec.mdk

```diff
@@ -679,6 +679,26 @@ described as follows:
       controller is determined to be a slave for a given (`device_id`,
       `role_id`).
 
+gRPC enables the server to identify which client originated each message in the
+`StreamChannel` stream. For example, the C++ gRPC library [@gRPCStreamC] in
+synchronous mode enables a server process to cause a function to be called when
+a new client creates a `StreamChannel` stream. This function should not return
+until the stream is closed and the server has done any cleanup required when a
+`StreamChannel` is closed normally (or broken, &eg; because a client process
+unexpectedly terminated). Thus the server can easily associate all
+`StreamChannel` messages received from the same client, because they are
+processed within the context of the same function call.
+
+A P4Runtime implementation need not rely on the gRPC library providing
+information with unary RPC messages that identify which client they came from.
+Unary RPC messages include requests to write table entries in the data plane, or
+read state from the data plane, among others described later. P4Runtime relies
+on clients identifying themselves in every write request, by including the
+values `device_id`, `role_id`, and `election_id` in all write requests. The
+server trusts clients not to use a triple of values other than their own in
+their write requests. gRPC provides authentication methods [@gRPCAuth] that can
+be deployed to prevent untrusted clients from making changes.
```

Comment:
from making establishing a channel actually, not just making changes

Reply:
clarified this point in the text in the next commit coming in a few minutes